### ISSUE TITLE
Friendship levels

### ISSUE THREAD

**Issue by d6stiny** [d6stiny/blursed#14]
### Checklist

- [X] I read and understood Blursed's **[Contributing guidelines](https://github.com/d6stiny/blursed/blob/main/CONTRIBUTING.md)**
- [X] I don't need **[help](https://github.com/d6stiny/blursed/discussions/categories/q-a)**, this is an enhancement idea
- [X] My idea doesn't duplicate existing Blursed functionality
- [X] I believe that my idea falls into Blursed's scope and should be offered as part of Blursed built-in functionality

### Enhancement purpose

basically the more you interact with your friends, the higher your friendship level becomes

### Solution

idk, we need to come up with a solution

### Why currently available solutions are not sufficient?

h

### Can you help us with this enhancement idea?

Yes, I can code the solution myself and send a pull request

### Additional info

(can code the backend)
